cmake/app/boilerplate.cmake

```diff
@@ -66,6 +66,8 @@ set_property(GLOBAL PROPERTY GENERATED_KERNEL_SOURCE_FILES "")
 
 set(APPLICATION_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR} CACHE PATH "Application Source Directory")
 set(APPLICATION_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR} CACHE PATH "Application Binary Directory")
+file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/.cmake/api/v1/query")
```

Comment:
Does this have to be here?  Feels like if nothing else a comment about what this is about.

Reply:
oops, good catch, i was testing something completely unrelated :)